starter-code/index.html

```diff
@@ -1,31 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
+  <div class="main-image">
+    <img class="phones" src=./images/phones.png>
+  </div>
+  <div class="main-logo">
+    <img id="logo" src=./images/brand.png>
+  </div>
+  <div class="main-button">
+    <button id="log">Log In</button>
+  </div>
 
-  Capture and Share the World's Moments
+  <div class="content">
+    <h2>Capture and Share <br> the World's Moments</h2>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+    <p>Instagram is a fast, beautiful and fun way to share your life with friends and family.</p>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+    <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
 
-  Oh yeah, did we mention it's free?
+    <p>Oh yeah, did we mention it's free?</p>
 
-  Download on the App Store
-  Get it on Google Play
+    <img id=download-ap src=./images/badge-iphone.png>
+    
+    <img id=download-an src=./images/badge-android.png>
+  </div>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+  <footer>
```

Comment:
If you inspect the footer, you'll notice that it is occupying more space than what it is supposed to do